modules/openthread/Kconfig.features

```diff
@@ -255,6 +255,24 @@ config OPENTHREAD_PLATFORM_KEY_REF
 	  Enable usage of cryptographic key references instead of literal keys.
 	  This requires a crypto backend library that supports key references.
 
+choice OPENTHREAD_PLATFORM_BOOTLOADER_MODE_CHOICE
+	prompt "Platform bootloader mode configuration"
+	optional
+
+config OPENTHREAD_PLATFORM_BOOTLOADER_MODE_SYS_REBOOT
+	bool "Bootloader mode support with sys reboot"
+	select OPENTHREAD_PLATFORM_BOOTLOADER_MODE
```

Comment:
This was deprecated some time ago, if this is for mcuboot then it should be using the boot mode retention API

Reply:
thanks! added retention API and dependencies. tested on nrf52840DK and sems to work fine